python/cudf/cudf/tests/test_concat.py

```diff
@@ -99,30 +101,45 @@ def test_concat_errors():
     df, df2, gdf, gdf2 = make_frames()
 
     # No objs
-    with pytest.raises(ValueError):
+    with pytest.raises(
+        ValueError, match="Need at least one object to concatenate"
+    ):
         gd.concat([])
```

Comment:
Instead of hardcoding the errors raised here can we run the equivalent functions in Pandas, catch the exception, and then ensure we're raising the same exception via cuDF?

Reply:
Changed similar cases across others tests too. Had to leave out few cases where our cudf errors were better in terms of error message or the types represented as part of error message differed(like `cudf.core.index.RangeIndex` vs `pandas.core.indexes.range.RangeIndex`)